README.md

```diff
@@ -5,8 +5,12 @@ Use Meteor's client side DDP protocol and Minimongo in a non Meteor project.
 
 ### Installation
 
+* Bower
 `bower install meteor-client-side`
 
+* NPM
+`npm install meteor-client-side-official`
```

Comment:
I think it is a bit wried to call it official when it is not **Meteor** official package